packages/node/src/indexer/indexer.manager.spec.ts

```diff
@@ -227,30 +227,28 @@ describe('IndexerManager', () => {
   });
 
   it.skip('should be able to start the manager (v0.0.1)', async () => {
-    indexerManager = createIndexerManager(
-      testSubqueryProject_1(),
-      new ConnectionPoolService<ApiPromiseConnection>(
-        nodeConfig,
-        new ConnectionPoolStateManager(),
-      ),
-      nodeConfig,
-    );
-    await expect(indexerManager.start()).resolves.toBe(undefined);
-
-    expect(Object.keys((indexerManager as any).vms).length).toBe(1);
+    // indexerManager = createIndexerManager(
+    //   testSubqueryProject_1(),
+    //   new ConnectionPoolService<ApiPromiseConnection>(
+    //     nodeConfig,
+    //     new ConnectionPoolStateManager(),
+    //   ),
+    //   nodeConfig,
+    // );
+    // await expect(indexerManager.start()).resolves.toBe(undefined);
+    // expect(Object.keys((indexerManager as any).vms).length).toBe(1);
   });
 
   it.skip('should be able to start the manager (v0.2.0)', async () => {
```

Comment:
I think we can remove these tests

Reply:
It would be good to bring back some tests for the indexer manager at some point so i left them commented